vars/defaults/archiveinterface.yml

```diff
@@ -0,0 +1,7 @@
+---
+pacifica_module: archiveinterface
+no_db: true
+socket_port: "8080"
+env_vars:
+   ARCHIVEINTERFACE_CONFIG: "/opt/default/archiveinterface.ini"
+   ARCHIVEINTERFACE_CPCONFIG: "/opt/default/archiveinterface-cp.ini"
```

Comment:
I could see an easy mismatch in fullpath to configs here. Not sure what the right fix is yet if any, just noting in case we need to consider other paths to auto reconfigure itself depending on defined virtualenv path per service.

Reply:
This goes for any services that pre-define a full path to the config (i.e. cartd_backend, cartd_frontend, etc.)

Reply:
This really needs to be fixed in the upstream packages, we'll need to add additional functionality to use default config files there first.